test/sys/test_aio.rs

```diff
@@ -433,20 +418,19 @@ lazy_static! {
 }
 
 extern fn sigfunc(_: c_int) {
-    SIGNALED.store(true, Ordering::Relaxed);
+    SIGNALED.store(true, Ordering::SeqCst);
```

Comment:
`Relaxed` is fine here, because `SIGNALED` isn't being used to control access to any other variables.  It's only being accessed by itself, so stricter atomic instructions aren't needed.  This is the kind of application that, in C, wouldn't even use special instructions, just a `sigatomic_t` variable.